app/.env.sample

```diff
@@ -26,6 +26,7 @@ REACT_APP_ACTION_UUID_NAMESPACE=35968df5-f335-44ae-8ae5-25804caa2d86
 REACT_APP_DEFAULT_PLAN_VERSION=1
 REACT_APP_ENABLE_TEAMS_ASSIGNMENT_MODULE=false
 REACT_APP_DEFAULT_PLAN_ID=35968df5-f335-44ae-8ae5-25804caa2d86
+REACT_APP_STAGE_ENVIRONMENT=eusm | tunisia
```

Comment:
I think its best to to have  it exactly as it would be in the `.env` to avoid confusion? Maybe `REACT_APP_STAGE_ENVIRONMENT=eusm` or `REACT_APP_STAGE_ENVIRONMENT=tunisia`

Reply:
Do we need to add documentation for this env in the env?

Reply:
@haseemVD lets use a less general env here given the scope of the change that is effected in the code. lets just use an env that targets the module that should be shown on the home route.

Reply:
@p-netm 

We can directly declare route, but when routes are changed we also need to change the value from env.
Not sure if its good approach

Reply:
hmm,.. that is a valid concern

Reply:
ok, can then we rename `STAGE_ENVIRONMENT` because  this will not apply for staging only.